spec/unit/product-manager.spec.js

```diff
@@ -130,4 +134,46 @@ describe('ProductManager', () => {
       ])
     })
   })
+
+  describe('compare two products', () => {
+    it('should equal when product ids are equal', () => {
+      productService = new ProductManager(utils.logger, {})
```

Comment:
This init of productService can be done in `beforeEach`?

Reply:
https://github.com/commercetools/commercetools-node-variant-reassignment/pull/30/commits/74990e7b1ab9e4d53c0d8574ef129156ce447a3b